### DISCUSSION TITLE
815

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#989] [Comments]
# 815

2021년 초에 합류했다가 벌써 만 4년이 다되어간다.올해는 국제화 기억만 있을 정도로 하나에만 완전히 몰입했다.아참, 올해 6월 1일에 결혼도 했다.그런데 막상 회고를 해보려고 하니 국제화 프로젝트 보다는 팀원들과 조직이 더 많이 생각났다.그래서 여기서는 회사에 관련된 주제들만 이야기할 예정이다.결혼, 트레바리 독서모임, 블로그, 책, 생활 등은 개인 회고로 정리할 예정이다.1. 조직1-1. 팀원올 한해 Product 조직에는 13명이 입사하고, 2명이 퇴사했다.입사: 올리(PD), 도니(FE), 하리(FE), 루시(FE), 럭끼(FE), 퍼즈(FE), 또우(Mobile), 감자(BE), 약풍(BE), 프레디(BE), 로켓(BE), 구피(DevOps), 포카(DevOps)퇴사: 홍시(FE), 하루(BE..

https://jojoldu.tistory.com/815

**Comment by my-yellow-star** [comment]
좋은 글 잘 읽었습니다.

**Comment by archan0621** [comment]
17년도부터 동욱님 글을 읽어서 어느새 5년차네요, 모든 글에서 항상 많이 배웁니다. 생각정리 글도 전체를 4번이나 재주행 했네요. 연말 잘 마무리하고 새해복 많이 받으세요!

**Comment by ljw1126** [comment]
결과보단 과정에 집중하고 이를 칭찬하고 글로 적은 걸 보면
팀원들도 동욱님한테 좋은 영향을 받고 다음으로 갔을거 같네요 
그리고 마지막에 스쳐지나갈 수 도 있었던, 웹툰 한 장면을 인용하신게 인상깊네요  
잘 보고 갑니다 
올 한해도 수고 많으셨습니다

**Comment by mag123c** [comment]
항상 느끼는거지만 조직 내에서 좋은 문화들을 만들어 나가신 덕에 선순환되는 것 같네요.
올 한해도 수고 많으셨고, 국제화가 잘 되었으면 좋겠습니다~ 수고하셨습니다!!!
내년에도 화이팅!

**Comment by icodetree** [comment]
향로님과 같이 영향력이 있는 분들은 왜 솔로프리너로 활동을 안할까 궁금증이 있었는데 글에서 답변을 해주셨네요~
스스로를 잘 알고 그에 맞는 방향 설정과 꾸준하게 성과를 내어가는 모습이 정말 귀감이 되는것 같습니다.
이런 행보가 정말 어려운거구나를 요즘 저도 아주 조금이지만 느끼고 있어 공감이 되네요.
웹툰의 이야기가 예전 제모습이 투영되는거 같아 여운이 남습니다.
공감되는 그리고 솔직한 이야기 잘읽고 갑니다.
올해도 건승하시길~

**Comment by bonomk2** [comment]
항상 글을 읽기만 하다 댓글 달아봅니다.
좋은 인재분들이 지속적으로 합류하는것이 왠지 부럽고 또 부럽습니다... ㅜㅜ
매해마다 안정기가 오기를 바라는데 그사이에 한해가 지나갔네요..

새해 건강하시고 생각하시는대로 잘 되길 바랄께요~ ^^
